src/pages/preferences/preferencesTabs/NewUITab.tsx

```diff
@@ -64,44 +64,47 @@ export const NewUITab: React.VFC = () => {
   };
 
   return (
-    <>
-      {/* @ts-expect-error */}
-      <PreferencesCard>
-        <PaddedBody>
-          Direct all inbound links to the new Evergreen UI, whenever possible
-          (e.g. from the CLI, GitHub, etc.).
-        </PaddedBody>
-        <Toggle
+    <PreferencesCard>
+      <PreferenceItem>
+        <StyledToggle
           checked={spruceV1}
           disabled={updateLoading}
           onChange={handleOnChangeNewUI}
           aria-label="Toggle new evergreen ui"
+          id="prefer-spruce"
+          size="small"
         />
-      </PreferencesCard>
-      {/* @ts-expect-error */}
-      <PreferencesCard>
-        <PaddedBody>
-          Allow background polling for active tabs in the current browser.
-        </PaddedBody>
-        <Toggle
+        <Label htmlFor="prefer-spruce">
+          Direct all inbound links to the new Evergreen UI, whenever possible
+          (e.g. from the CLI, GitHub, etc.).
+        </Label>
+      </PreferenceItem>
+      <PreferenceItem>
+        <StyledToggle
           checked={Cookies.get(DISABLE_QUERY_POLLING) !== "true"}
           onChange={handleOnChangePolling}
-          aria-label="Toggle new evergreen ui"
+          aria-label="Toggle background polling"
+          id="polling"
+          size="small"
         />
-      </PreferencesCard>
-    </>
+        <Label htmlFor="polling">
+          Allow background polling for active tabs in the current browser.
+        </Label>
+      </PreferenceItem>
+    </PreferencesCard>
   );
 };
 
 // @ts-expect-error
-const PreferencesCard = styled(Card)`
-  display: flex;
-  flex-direction: column;
-  padding: ${size.m};
-  margin-bottom: ${size.m};
-  width: 100%;
+const StyledToggle = styled(Toggle)`
+  margin-right: ${size.xs};
 `;
 
-const PaddedBody = styled(Body)`
-  padding-bottom: ${size.l};
+const PreferenceItem = styled.div`
+  align-items: center;
+  display: flex;
+
```

Comment:
I think you can remove `StyledToggle` and just put a `gap` on `PreferenceItem`

Reply:
ty Miss Gap